src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/EventType.java

```diff
@@ -125,7 +125,8 @@
     FE_NO_ACTION("feNoAction", 0, false),
     SEND_TO_ADMIN("sendToAdmin", 0, false),
     READY_TO_LIST("readyToList", 0, false),
-    DWP_DIRECTION_RESPONSE("dwpDirectionResponse", 0, false);
+    DWP_DIRECTION_RESPONSE("dwpDirectionResponse", 0, false),
+    VALID_SENT_TO_INTERLOC("validSendToInterloc", 0, false);
```

Comment:
Minor point but shouldn't this be `VALID_SEND_TO_INTERLOC` instead of `VALID_SENT_TO_INTERLOC`